egs/wsj/s5/steps/nnet3/make_bottleneck_features.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+#           2016  Pegah Ghahremani
+# Apache 2.0
+# This script dumps bottleneck feature for model trained using nnet3.
+
+# Begin configuration section.
+stage=1
+nj=4
+cmd=run.pl
+use_gpu=false
+bnf_name=Tdnn_Bottleneck_renorm
```

Comment:
The whole point is that this script can be used to dump output of different layers not only bottleneck. That was the reason I didn't fix the name!